src/styles/blocks/page.scss

```diff
@@ -0,0 +1,34 @@
+.page {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
```

Comment:
font-weight: 700 are duplicated in other places, so please move this and other duplicated styles to variables